tc/autotuner/utils.cc

```diff
@@ -121,21 +115,19 @@ void Printer::stop() {
 }
 
 void Printer::printAll() {
-  auto runtimes = [this]() {
+  auto getSortedRuntimes = [this]() {
     std::lock_guard<std::mutex> lock(runtimesMtx_);
     std::sort(runtimes_.begin(), runtimes_.end());
```

Comment:
not sure why this sorts the internal `runtimes_` list each time this function called, under lock, just to _copy_ them to another vector for the sake of output

Reply:
There's no need to. Should be: 

1. copy under lock
2. release lock
3. sort local and print

Reply:
legacy I'll give it a quick shot but if I run into issues I'll open a task for @ttheodor  to clean up